### DISCUSSION TITLE
Hyprland has become independent from wlroots, can I still use wayvnc?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [hyprwm/Hyprland#7634] [General]
I use hyprland in a headless environment, and wayvnc is the only headless vnc available in wayland. But wayvnc only supports wlroots-based wm. Does that mean I won't be able to use wayvnc?

**Comment by vaxerski** [comment]
no, should still work

**Reply by fxzxmicah** [reply]
I can now confirm that hyprland 0.46.2 is no longer compatible with wayvnc (A smaller possibility is that headless mode is no longer compatible with wayvnc).
When I connect to wayvnc it just shows a completely grey screen.

Before that, I just updated hyprland from 0.45.2 to 0.46.2, as well as installing the new dependencies added in 0.46.2.

**Reply by fxzxmicah** [reply]
https://github.com/hyprwm/Hyprland/issues/8806 Oh, I found it.
